trp_project_scrum/tests/test_trp_project_scrum.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+from openerp.exceptions import AccessError
+import random
+import string
+
+
+class TestTrpProjectScrum(TransactionCase):
+
+    def setUp(self):
+        super(TestTrpProjectScrum, self).setUp()
+        # will trigger implied IDS of other groups
+        # let's make believe we are logged in by passing 'login' in data
+        # create a random login just in case we get one that already exists
+        login1 = ''.join(
```

Comment:
You could alternatively create the collaborator user as demo data. The big advantage is that we then also can use this for front end testing and ofcourse for demo's. I would suggest:
name: Demo Scrum Collaborator
login: demo_collaborator
password: demo_collaborator